packages/react/src/components/IconButton/index.tsx

```diff
@@ -16,7 +16,7 @@ interface IconButtonProps
   tooltipPlacement?: TooltipProps['placement'];
   tooltipVariant?: TooltipProps['variant'];
   tooltipPortal?: TooltipProps['portal'];
-  variant?: 'light' | 'dark' | 'primary' | 'secondary' | 'error';
```

Comment:
This is a breaking change, so ensure that the body of the pr includes `BREAKING CHANGE:` with a description of what is breaking.